doc/moveit_grasps/moveit_grasps_tutorial.rst

```diff
@@ -140,7 +140,16 @@ The result should look something like this:
 
 3) Visualize Grasp Generation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-To visualize grasp generation for randomly placed blocks::
+
+This tool demonstrates several concepts around how scoring works during grasp generation.
+It bases the chosen grasp using the following approaches:
+
+ - *Ideal TCP Grasp Pose*: the generator will bias the chosen grasp based on an input of preferred grasp. This allows you to say, for example, 'I want it to be as close as possible to a grasp that is dead-ceneted on the object'.
```

Comment:
I think the ideal tcp grasp pose is only used for orientation. The translation only gets used for visualization

Reply:
The ideal grasp pose for suction is at the middle of the object, for fingers I believe it is calculated to be the middle of the grasp candidates

Reply:
Can you make a suggested change of what you mean here?